app/config/featureToggles/FeatureSwitch.scala

```diff
@@ -23,9 +23,7 @@ object FeatureSwitch {
   val featureSwitches: Seq[FeatureSwitch] = Seq(
     AccessMTD,
     IvUpliftSwitch,
-    PptSwitch,
```

Comment:
Seems to be all checks passed. so I do not think ATs would have impacted. Please check All ATS are okay or not?

Reply:
All ATs are checked and resulted in no failure.